.gitignore

```diff
@@ -1,4 +1,64 @@
 
+## PureBasic
+############
+*.pb.cfg
+*.pbi.cfg
+*.pbp
+project.cfg
+
+
+## Binary files
+###############
+
+# Object files
+# ------------
+*.elf
+*.ko
+*.o
+*.obj
+
+# Static libraries
+# ----------------
+*.a
+*.la
+*.lib
+*.lo
+
+# Shared objects
+# --------------
+*.dll
+*.dylib
+*.so
+*.so.*
+
+# Executables
+# -----------
+*.app
+*.exe
+*.out
+
+## Misc Work Files
+##################
+
+# Temporary/backup or removed files and folders:
+___*
+___*.*
+
+# Links and URLs
+*.lnk
+*.url
+
+
+## Apps Local Settings & Temp Files
+###################################
+
+# Chrome's debug logs for local HTML pages:
+debug.log
+
+############################
+## COMMON IGNORE PATTERNS ##
+############################
+
 # Created by https://www.toptal.com/developers/gitignore/api/windows,linux,macos,purebasic
 # Edit at https://www.toptal.com/developers/gitignore?templates=windows,linux,macos,purebasic
```

Comment:
The source information is then misplaced here, should be removed and should rather be placed at the beginning of the file. For example:
> Content largely based on: https://www.toptal.com/developers/gitignore/api/windows,linux,macos,purebasic

Reply:
No, that would be inaccurate since it would attribute patterns that I have created to the source link, whereas right now some patterns are missing, but its fine (everything from the link below is still taken verbatim from the source doc).

Reply:
Ok